keyboards/mechlovin/infinity87/rgb/rev1/info.json

```diff
@@ -0,0 +1,15 @@
+{
+    "processor": "STM32F303",
+    "bootloader": "stm32-dfu",
+    "board": "QMK_PROTON_C",
```

Comment:
Is this actually using a proton C?

Reply:
Nope, I just move from infinity87/rgb_rev1.
I will update, thank you.